components/layout_2020/formatting_contexts.rs

```diff
@@ -73,6 +73,11 @@ pub(crate) struct IndependentLayout {
     /// <https://drafts.csswg.org/css2/visudet.html#root-height>
     pub content_block_size: Au,
 
+    /// The contents of a table may force it to become wider than what we would expect
+    /// from 'width' and 'min-width'. This is the resulting inline content size,
+    /// or None for non-table layouts.
+    pub content_inline_size_for_table: Option<Au>,
```

Comment:
It might be worth adding this text from the specification here:

```
    /// > See <https://drafts.csswg.org/css-tables/#layout>:
    /// > Unlike other block-level boxes, tables do not fill their containing block by default. 
    /// > When their [width](https://drafts.csswg.org/css-sizing-3/#propdef-width) computes
    /// > to auto, they behave as if they had fit-content specified instead. This is different from 
    /// > most block-level boxes, which behave as if they had stretch instead.
```